### DISCUSSION TITLE
VipsForeignLoad image is not a known file format

### DISCUSSION THREAD

**Discussion by ChunAllen** [libvips/ruby-vips#293] [Q&A]
I'm currently using **vips-8.10.6** and installed it in my EC2 CentOS server. 

When I vips--version I have this 
```
vips-8.10.6-Mon Mar  8 15:23:31 UTC 2021
```

I have a jpeg file under 
```
zipped_transactions/034120210329172400590_imgRef.jpeg
```

When I try 
```
Vips::Image.new_from_file 'zipped_transactions/034120210329172400590_imgRef.jpeg', access: sequential
```

I'm getting 
```
VipsForeignLoad image is not a known file format
```

From my local when I use the same image I can get the average of the image

**Comment by tsrivishnu** [comment]
Seems like the required dependancies/libraries are not installed. Maybe this helps: https://github.com/janko/image_processing/issues/97#issuecomment-1019520835
